scipy/spatial/_plotutils.py

```diff
@@ -153,6 +153,32 @@ def voronoi_plot_2d(vor, ax=None, **kw):
     -----
     Requires Matplotlib.
 
+    Examples
+    --------
+    Set of point:
+
+    >>> import numpy as np    
+    >>> import matplotlib.pyplot as plt
+    >>> points = np.random.rand(10,2) #random
+    >>> plt.scatter(points[:,0],point[:,1])
+    >>> plt.show()
+
+    Voronoi diagram of the points:
+
+    >>> from scipy.spatial import Voronoi, voronoi_plot_2d
+    >>> vor = Voronoi(points)
+
+    using voronoi_plot_2d for visualisation:
+
+    >>> voronoi_plot_2d(vor)
+    >>> plt.show()
+
+    using voronoi_plot_2d for visualisation with enhancements:
+
+    >>> voronoi_plot_2d(vor, show_verices=False, line_color='orange',\
+            line_width=2, line_alpha=0.6, point_size=2)
```

Comment:
This results in a CI failure: https://travis-ci.org/scipy/scipy/jobs/350021128

You can resolve that by doing ``fig = voronoi_plot_2d(...``

Reply:
Actually there's a couple of failures here, please have a detailed look at the CI output.

Reply:
typo `verices` should be `vertices`

Reply:
Also no ``\`` please, it's better never to use those in Python code. In this case, just remove the ``\`` and below the `>>>` put `...` which means line continuation.

Reply:
Thank @rgommers for the review. What should I use instead of '\' for long lines? or is it not needed at all?